### DISCUSSION TITLE
REAL-TIME STREAMING HACKATHON (2023/Q1) 🔔

### DISCUSSION THREAD

**Discussion by umtcylmz** [ant-media/Ant-Media-Server#4848] [Announcements]
## ℹ️ About the Challenge 
This is a call for developers all around the world to join us in this hackathon and create interactive applications or plugins using the Ant Media Server and win up to **$6000** in Prizes! 🚀 

You will receive a starter kit after you completed the registration process. Also, Ant Media experts will support you during the Hackathon. 

Use your creativity and put together great interactive streaming mobile or web applications/plugins about any topic you may like such as chatting, IoT, data, and gaming applications, the sky is the limit! **[Register Now!](https://docs.google.com/forms/d/e/1FAIpQLSffPtG-k-dmDIUH-7f5yV0nm80H-RLvBefu9hrSCLlNPjO1tA/viewform?usp=share_link)**

## 🏆 Prize Pool 
**🥇 1st prize: $3000 
🥈 2nd prize: $2000
🥉 3rd prize: $1000**

There will be mini-challenges with special gifts! **[Register Now!](https://docs.google.com/forms/d/e/1FAIpQLSffPtG-k-dmDIUH-7f5yV0nm80H-RLvBefu9hrSCLlNPjO1tA/viewform?usp=share_link)**

## 📱 Marketplace
If you successfully finish the development process and create a product at the end, on [Ant Media marketplace](https://antmedia.io/marketplace/) you will have the opportunity to promote and also sell it **[Register Now!](https://docs.google.com/forms/d/e/1FAIpQLSffPtG-k-dmDIUH-7f5yV0nm80H-RLvBefu9hrSCLlNPjO1tA/viewform?usp=share_link)**

## 📆 Agenda

| Event | Date |
|---|---|
| ⏺ Challenge introduction  | 28th of February |
| 📨 Registration process starts | 1st of March |
| 📹  Live session – Pre-event | 26th of March at 16:00 CET |
| 🚀 Hackathon period | 1st of April from 9:00 CET through 2nd of April 12:00 CET |
| 🚩 First milestone upload | 1 April @ 11:00 (final presentation slides) |
| 🚩 Second milestone upload | 2 April @ 11:30 (war, ipa, apk files upload) |
| 🛑 Project submission deadline | 2nd of April  11:59 CET |
| 📹 Demo presentations & judging period | 2nd of April from 15:00 CET |
| 🏆 Award ceremony | 2nd of April at 17:00 CET |

##  🎯  Starter Kit

[Here](https://drive.google.com/file/d/11ovdDam99Pela3AkP0fvGCQyb0gZHXBZ/view?usp=share_link) you can download the starter kit, explore and carefully read the Starter Kit. The document provides you with all the information you need to participate in this hackathon, such as:

- Hackathon's goal
- Topics & Challenges description
- Event detailed agenda and deadlines
- Description of the deliverables you should submit during the Hackathon
- Link to the Resources Kit
- Judging period and evaluation criteria
- Winners announcement procedure

**Comment by asharamseervi** [comment]
We're excited to share that our Real-time Streaming Hackathon((2023/Q1)) was a huge success! 🎉

We had received 76 amazing participants registration, including recognized startups, contributors, and members from our open-source communities.

And, We would like to recognize winners with a big round of applause for the Real-Time Streaming Hackathon (2023/Q1):
🥁🥁🥁👏👏👏

🥇 **First Place:**  _[Muhammad Usman Bashir](https://www.linkedin.com/in/muhammdusmanbashir786/)_ from **RTC Mavericks**
🥈 Second Place: _[Richárd Ádám Dr. Vécsey](https://www.linkedin.com/in/richard-adam-vecsey-dr/)_ and _[Axel Ország-Krisz Dr.](https://www.linkedin.com/in/axel-orszag-krisz-dr/)_ from **Twiddle**
🥉 Third Place: _[Meghraj Suthar](https://www.linkedin.com/in/imromec/)_ from **Growcify**

A massive shoutout to all our participants for their hard work and dedication. The competition was fierce, and the creativity and innovation blew us away. Checkout our blog post for a complete overview of the Hackathon: https://antmedia.io/what-happened-in-the-first-hackathon-of-the-year-ant-media-will-organize-3-more-hackathons-in-2023/

We also want to thank our judges, mentors, and everyone from our team members, for supporting this event and helping us make it possible. Your contributions have made this hackathon a memorable experience for everyone involved.

Thank you again to everyone who participated, and let's keep the momentum going! 💪

See you at Real-time Streaming Hackathon (2023/Q2).
Stay tuned for more exciting updates from Ant Media! 🎉
